mobile/test/java/org/chromium/net/CronetHttp3Test.java

```diff
@@ -40,7 +40,7 @@ public class CronetHttp3Test {
 
   // If true, dump envoy logs on test completion.
   // Ideally we could override this from the command line but that's TBD.
-  private boolean printEnvoyLogs = true;
+  private boolean printEnvoyLogs = false;
```

Comment:
should we keep this as true?  then we'd get more useful info on test failures

Reply:
I think initially this was set to false, but I agree that true is actually better. Let me fix it.

Reply:
let's keep off by default and enable if tests get flaky

Reply:
Done. Setting it back to `false`.

Reply:
haha, either way really